ssh_check/tests/test_ssh.py

```diff
@@ -0,0 +1,80 @@
+# (C) Datadog, Inc. 2018
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import threading
+
+import pytest
+from datadog_checks.stubs import aggregator
+
+from datadog_checks.ssh_check import CheckSSH
+
+
+@pytest.fixture
+def Aggregator():
+    aggregator.reset()
+    return aggregator
+
+
+class TestSshCheck:
```

Comment:
Why do you need a class here? Since `pytest` doesn't require a function to collect test cases, I think a class here only gives you unnecessary indentation and the need for `self`

Reply:
I can change this if you want.

I've always grouped the testing of like functionality into classes. For example, this check also does sftp, which might be grouped together in another class. Having a bunch of functions whose functionality is only organized by naming always bothered me.

Reply:
you can use modules to break down / group free functions (also valid when you end up with too many classes in the same module)